src/components/Inputs/CustomLookup.js

```diff
@@ -23,15 +23,15 @@ const CustomLookup = ({
   onLookup,
   onChange,
   error,
-  firstFieldWidth = '210px',
+  firstFieldWidth = '100%',
   helperText,
   variant = 'outlined', //outlined, standard, filled
   size = 'small', //small, medium
   required = false,
   autoFocus = false,
   disabled = false,
   readOnly = false,
-  editMode,
+  editMode, dataGrid=false,
```

Comment:
Why the datagrid prop? Name it 'hasBorder' instead and set it to false by default